x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -79,7 +80,22 @@ export function StickyTransactionProperties({
           defaultMessage: '% of trace'
         }
       ),
-      val: asPercent(duration, totalDuration, NOT_AVAILABLE_LABEL),
+      val:
+        totalDuration !== undefined && duration > totalDuration ? (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.apm.transactionDetails.percentOfTraceLabelExplanation',
+              {
+                defaultMessage:
+                  'The % of trace exceeds 100% because this transaction takes longer than the root transaction.'
+              }
+            )}
+          >
+            <span>&gt;100%</span>
```

Comment:
does `'>100%'` need to be localized as well?

Reply:
Hmm, maybe. I've asked the i18n folks for clarification.